charts/immich/values.yaml

```diff
@@ -14,7 +14,8 @@ env:
   IMMICH_MACHINE_LEARNING_URL: '{{ printf "http://%s-machine-learning:3003" .Release.Name }}'
 
 image:
-  tag: v1.119.0
+  repository: ghcr.io/immich-app/immich-server
```

Comment:
this was necessary to get renovate to auto update, or rather auto open PRs when updates are available